src/tests/EdFi.DataManagementService.Tests.E2E/Features/Descriptors/ReadDescriptorsValidation.feature

```diff
@@ -52,16 +50,19 @@ Feature: Resources "Read" Operation validations
                     }
                   """
 
-        @ignore
         Scenario: 03 Verify response code 404 when ID does not exist
-             When a GET request is made to "/ed-fi/absenceEventCategoryDescriptors/123123123123"
```

Comment:
This is an invalid ID. We need to distinguish between an ID that does not exist and one that is not valid. There is a new scenario at the bottom of the file for invalid ID.